integration/test-data/test-stub/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+// Test Stub to use as workload stand-in in integration tests.
+func main() {
+	fmt.Fprintln(os.Stdout, "waiting for something interesting to happen...")
+
+	// block forever
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
the `syscall` pkg is deprecated. Use `os.Interrupt` insted of `syscall.SIGINT` and `syscall.SIGTERM`

Reply:
Thanks for the catch! Should be better now.